spec/features/past_chancellors_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "integration_spec_helper"
+
+RSpec.feature "Past Chancellors pages" do
+  let(:base_path) { "/government/history/past-chancellors" }
+  let(:content_item) do
+    {
+      base_path:,
+      title: "Past Chancellors of the Exchequer",
+    }
+  end
+
+  before do
+    stub_content_store_has_item(base_path, content_item)
+  end
+
+  it "sets the page title" do
+    visit base_path
+    expect(page).to have_title("#{content_item['title']} - GOV.UK")
```

Comment:
Just noticed this as I was copying these tests for mine - title here (and a few lines below) should be a symbol

Reply:
Thanks.  Test passed anyway, so the hash must be using indifferent access.  I'll update to a symbol for consistency.

Reply:
In case of interest, that's what I had thought but actually I noticed this I copied it but the test didn't fail when I expected it to. Turns out it's because of the string interpolation - interpolating a nil gets you an empty string, so: 
`puts "#{content_item['title']} - GOV.UK"`
prints 
` - GOV.UK` 
so I had a passing test because e.g. the title always has `- GOV.UK` at the end

Reply:
Oh, that makes sense.  I hadn't realised that was happening!